src/pages/Account/Tabs/ModulesTab/Contract.tsx

```diff
@@ -695,8 +695,6 @@ function ReadContractForm({
   const aptosClient = useAptosClient();
   const sdkV2Client = useSdkV2Client();
   const [result, setResult] = useState<Types.MoveValue[]>();
-  const theme = useTheme();
-  const isWideScreen = useMediaQuery(theme.breakpoints.up("md"));
```

Comment:
Interesting, why get rid of the existing theme stuff?

Reply:
Originally, I thought it would be better to keep the same layout structure, but after taking another look, I decided to revert to the previous approach: 

- on desktop, the result and the copy button are on the same row
- on mobile, the copy button is placed below the result.